nominal/nominal.py

```diff
@@ -339,3 +348,41 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+def create_checklist(
+    assignee_email: str,
```

Comment:
preference to put this right before the args with defaults so that we can eventually turn this optional - i don't think this should need to be required in the future

Reply:
agreed